.github/workflows/reusable_gemini_review.yml

```diff
@@ -0,0 +1,71 @@
+name: Reusable Gemini AI Code Review
+
+# ------------------------------------------------------------------------------
+# This workflow is only run when explicitly called by another workflow.
+# Please make sure the related secrets have been configured in your GitHub repo.
+# ------------------------------------------------------------------------------
+on:
+  workflow_call:
+
+# This workflow is passive so ensure the minimal permissions even if the
+# inherited token is more lenient.
+permissions:
+  contents: read
+  pull-requests: none
+
+jobs:
+  review:
+    if: >-
+      ${{ !contains(github.event.pull_request.labels.*.name, 'dependencies') &&
+      contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.pull_request.author_association) }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write # Required to post comments
+
+    steps:
+      - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+
+      # 1. Install Node.js
+      - uses: actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903 # v6.0.0
+
+      # 2. Install Gemini CLI
+      - name: Install Gemini CLI
+        run: npm install -g @google/gemini-cli
+
+      # 3. Authenticate to Google Cloud (The Safe Way)
+      - id: 'auth'
+        name: 'Authenticate to GCP'
+        uses: 'google-github-actions/auth@7c6bc770dae815cd3e89ee6cdf493a5fab2cc093' # v3.0.0
+        with:
+          credentials_json: '${{ secrets.GCP_SA_KEY }}'
```

Comment:
The secret could be a required input informed by the caller for more flexibility. WDYT?

Reply:
@marcusburghardt Thank you for your review. I updated it.